### DISCUSSION TITLE
How can I tell if my attempts to flash klipper have succeeded and/or why they've failed?

### DISCUSSION THREAD

**Discussion by ianloic** [bassamanator/everything-sovol-sv06#31] [Q&A]
I got an SV-06 last week running stock factory marlin, but I want to run Klipper on it. I've tried flashing klipper firmware, both that I built myself and that I downloaded from others. I've used a wide variety of micro sd cards, mostly SanDisk. I've formatted the disks on Linux and on Windows.

So far Klipper always gives me errors about timeouts waiting for identify_response. The UI on the printer's LCD screen is identical to before I tried flashing. Default octoprint keeps printing great. The response to `M115` is:
```
Send: M115
Recv: FIRMWARE_NAME:Marlin 2.0.9.2 (Feb 24 2023 09:21:18) SOURCE_CODE_URL:github.com/MarlinFirmware/Marlin PROTOCOL_VERSION:V2.0.0_A MACHINE_TYPE:SOVOL SV06 EXTRUDER_COUNT:1 UUID:cede2a2f-41a2-4748-9b12-c55c62f367ff
Recv: Cap:SERIAL_XON_XOFF:0
Recv: Cap:BINARY_FILE_TRANSFER:0
Recv: Cap:EEPROM:1
Recv: Cap:VOLUMETRIC:1
Recv: Cap:AUTOREPORT_POS:0
Recv: Cap:AUTOREPORT_TEMP:1
Recv: Cap:PROGRESS:0
Recv: Cap:PRINT_JOB:1
Recv: Cap:AUTOLEVEL:1
Recv: Cap:RUNOUT:0
Recv: Cap:Z_PROBE:1
Recv: Cap:LEVELING_DATA:1
Recv: Cap:BUILD_PERCENT:0
Recv: Cap:SOFTWARE_POWER:0
Recv: Cap:TOGGLE_LIGHTS:0
Recv: Cap:CASE_LIGHT_BRIGHTNESS:0
Recv: Cap:EMERGENCY_PARSER:0
Recv: Cap:HOST_ACTION_COMMANDS:1
Recv: Cap:PROMPT_SUPPORT:0
Recv: Cap:SDCARD:1
Recv: Cap:REPEAT:0
Recv: Cap:SD_WRITE:1
Recv: Cap:AUTOREPORT_SD_STATUS:0
Recv: Cap:LONG_FILENAME:1
Recv: Cap:THERMAL_PROTECTION:1
Recv: Cap:MOTION_MODES:0
Recv: Cap:ARCS:1
Recv: Cap:BABYSTEPPING:1
Recv: Cap:CHAMBER_TEMPERATURE:0
Recv: Cap:COOLER_TEMPERATURE:0
Recv: Cap:MEATPACK:0
Recv: ok
Send: M155 S2
Recv: ok
```

So I'm pretty sure that my Klipper flashes have failed. Anyone have any ideas or tricks for getting them to work?

Failing that I'm not beyond hooking up debug probes to the motherboard, dumping / modifying the eeprom, etc. I've done plenty of professional and recreational embedded systems development and debugging.

**Comment by bassamanator** [comment]
You're on the wrong repo, my friend, but I suppose you're in the ballpark.

Check my other repo for instructions.
